src/styles/blocks/catalog.scss

```diff
@@ -11,12 +10,17 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
+    }
```

Comment:
you need to set not only duration but with styles that will be modified by the transition. Like `transition: color , transform`  [read](https://developer.mozilla.org/ru/docs/Web/CSS/transition)